efficientdet/aug/autoaugment_test.py

```diff
@@ -26,10 +26,12 @@
 class AutoaugmentTest(tf.test.TestCase):
 
   def test_autoaugment_policy(self):
+    tf.disable_eager_execution()
```

Comment:
Since tf.disable_eager_execution() has already run before tf.test.main(), you probably don't need tf.disable_eager_execution() here?